tox_dist.ini

```diff
@@ -15,6 +15,7 @@ envlist = py35, py36, py37, py38
 passenv = RDIFF_TEST_* RDIFF_BACKUP_*
 skip_install = true
 deps =
+    setuptools-scm
```

Comment:
Any reason why `setuptools-scm` vs `setuptools` ?

Reply:
Not sure it was required, but it's because the runtime of this fox isn't rdiff-backup but the wheel packaging of it.